src/app/helptext/jails/jails-add.ts

```diff
@@ -543,6 +552,12 @@ rtsold_tooltip: T('Use <a href="https://www.freebsd.org/cgi/man.cgi?query=rtsold
 target="_blank">rtsold(8)</a> as part of IPv6 autoconfiguration. \
 Send ICMPv6 Router Solicitation messages to interfaces to discover new routers.'),
 
+ip_hostname_placeholder: T('ip_hostname'),
+ip_hostname_tooltip: T(''),
```

Comment:
Set to use DNS records during jail IP configuration to search the resolver and apply the first open IPv4 and IPv6 addresses. See <a href="https://www.freebsd.org/cgi/man.cgi?query=jail" target="_blank">jail(8)</a>.